instrumentation/rmi/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/rmi/context/ContextPayload.java

```diff
@@ -24,12 +22,6 @@
 
   private static final Logger log = LoggerFactory.getLogger(ContextPayload.class);
 
-  private static final Tracer TRACER = GlobalOpenTelemetry.getTracer("io.opentelemetry.auto.rmi");
```

Comment:
this is a good change, but I don't see how it's related to the topic of the PR.

Reply:
It was just a bit of unused code - I thought that removing it in a separate PR would be a bit of excess of form, but you're absolutely right that it't not related to this PR.